client_cli/README.md

```diff
@@ -43,71 +49,52 @@ Full description and list of commands detailed in `./iroha_client_cli --help`.
-For domain entity we only need `id` parameter as a string.
+Let's start with domain creation. To create a domain, you need to specify the entity type first (`domain` in our case) and then the command (`register`) with a list of required parameters.
+For domain entity you only need to provide the `id` parameter as a string.
 
 ```bash
 ./iroha_client_cli domain register --id="Soramitsu"
 ```
 
+Now you have a domain without any accounts.
+
 ### Create new Account
 
-Right now we have the only domain without any accounts, let's fix it.
-Like in the previous example, we need to define account name, it is done using `id` flag.
-We also give a `key` argument with account's public key as a double-quoted
-string value.
+Let's create a new account. Like in the previous example, specify the entity type (`account`) and the command (`register`). Then define the account name with the `id` flag.
+You also need to provide the `key` argument with the account's public key as a double-quoted
+string.
 
 ```bash
 ./iroha_client_cli account register --id="White Rabbit@Soramitsu" --key=""
 ```
 
 ### Mint Asset to Account
 
-Okay, it's time to give something to our Account. We will add some Assets quantity to it.
-This time we need to register an Asset Definition first and then add some Assets to the account.
-As you can see, we use new command `asset` and it's subcommands `register` and `mint`. 
-Every asset has its own value type, here we define domain as quantity (integer/number).
+It's time to give something to the Account you created. Let's add some Assets quantity to it.
+
+This time, you need to register an Asset Definition first and then add some Assets to the account. To do this, specify the `asset` entity and then use the `register` and `mint` commands respectively. Every asset has its own value type. In this example, it is defined as quantity (integer/number).
 
 ```bash
 ./iroha_client_cli asset register --id="XOR#Soramitsu" --value-type=Quantity
 ./iroha_client_cli asset mint --account="White Rabbit@Soramitsu" --asset="XOR#Soramitsu" --quantity=1010 
 ```
 
+You created `XOR#Soramitsu` asset of type `Quantity` and then gave `1010` units of this asset to the `White Rabbit@Soramitsu` account.
+
 ### Query Account Assets Quantity
 
 Because distributed systems heavily relay on the concept of eventual consistency and Iroha works in Consensus between peers, your requests may or may not be processed
-while Iroha Client will successfully send them and Iroha Peer will accept them. Different stages of transactions processing and different cases may lead to
-rejection of transaction after your receive response from Command Line Interface. To check that your instruction were applied and system now in the desired state
-you need to become familiar and use Query API.
+while Iroha Client will successfully send them and Iroha Peer will accept them. Different stages of transaction processing and different cases may lead to
```

Comment:
```suggestion
While the Iroha Client will successfully send your transactions and the  Iroha Peer will confirm receiving them, it is possible that your request will not appear in the next block. Different causes such as a transaction timeout, a faulty peer in the network, catastrophic failure of the peer that you've sent your data towards and many other conditions naturally occurring inside of any blockchain may lead to a rejection of your transaction at many different stages of processing. It should be noted that Iroha is designed to reduce the incidence of such rejections, and only rejects properly formed transactions in situations when not rejecting it would lead to data corruption and a hard-fork of the network. 
```